syntax = "proto3";

package payment;

message PaymentNotificationProto {
    int32 order_id = 1;
    float amount = 2;
    string currency = 3;
    string payment_method = 4;
}
